extensions/amp-story/1.0/test/test-amp-story-grid-layer.js

```diff
@@ -57,7 +57,9 @@ describes.realWin('amp-story-grid-layer', {amp: true}, (env) => {
     });
 
     const story = win.document.createElement('amp-story');
+    // Makes whenUpgradedToCustomElement() resolve immediately.
```

Comment:
I think this line belongs on top of `story.createdCallback = Promise.resolve();`